### ISSUE TITLE
Job execution fails with version 2.0.6

### ISSUE THREAD

**Issue by mirkoknoll** [rundeck-plugins/py-winrm-plugin#50]
Hi, 
we have configured one Windows host in Rundeck project with ssh default node executor via resource.xml 

` <node name="myhost" node-executor="WinRMPython" hostname="myhost"  winrm-transport="https" osFamily="Windows" winrm-authtype="basic" winrm-nossl="true" winrm-port="5986" winrm-password-storage-path="mykeypath" username="myuser" /> `

 and job execution was working with Rundeck 3.2.4 and py-winrm-plugin 2.0.5.
On debug output I get this:

`[WinRMPython] executing: [[/bin/sh, /opt/rundeck/libext/cache/py-winrm-plugin-2.0.5/wrapper.sh, winrm-exec.py, ${node.hostname}]] `

The same config is also working with Rundeck 3.2.5 and 3.2.6 and py-winrm-plugin 2.0.5.

With Rundeck 3.2.5 or 3.2.6 and py-winrm-plugin 2.0.6 I get this:
`[WinRMPython] executing: [[${config.interpreter}, -u, /opt/rundeck/libext/cache/py-winrm-plugin-2.0.6/winrm-exec.py, ${node.hostname}]] `
and the job is failing with:
`[ERROR  ]  winrm not installed, try: pip install pywinrm (winrm-exec.py:187)[root] `

Do I have to set config.interpreter somewhere in job or Rundeck config? 

On Rundeck server itself (Debian) I can execute winrm-check.py successfully:

`/opt/rundeck/libext/cache/py-winrm-plugin-2.0.6$ python winrm-check.py --username myuser --hostname myhost --password 'mypw' --port 5986 --transport https
b'\r\nWindows IP Configuration\r\n\r\n\r\nEthernet adapter Prod:\r\n\r\n   Connection-specific DNS Suffix  . : mydomain\r\n   IPv4 Address. . . . . . . . . . . : 10.11.12.13\r\n   Subnet Mask . . . . . . . . . . . : 255.255.192.0\r\n   Default Gateway . . . . . . . . . : 10.11.12.1\r\n'
Connection with host myhost successfull `

Cheers
Mirko

**Comment by master-qwerty** [comment]
Hello,
I have the same issue. Is there anyone who solved it?
`[ERROR  ]  winrm not installed, try: pip install pywinrm (winrm-exec.py:224)[root]`
Project is configured : **Default Node Executor > Python Interpreter = python3**
A job with workflow '_WinRM Check Step_' has passed successfully.

Versions:
- Rundeck (ver. 3.3.8-20210111 2021-01-11) 
- Ubuntu 20.04.1 LTS
- Target: Windows Server 2019

**Comment by Samaruq** [comment]
Having the same problem as @master-qwerty.

`winrm-check` from local Rundeck server to remote Windows node succeeds but Job execution to that remote node fails with `[ERROR  ]  winrm not installed, try: pip install pywinrm (winrm-exec.py:224)[root]`.

**Comment by Samaruq** [comment]
I have just solved my version of this problem.
As specified in the plugin docs I installed the `winrm` module using `pip` with
```console
$ pip install pywinrm
```
but the `winrm-exec.py` script was unable to find the module. (while `winrm-check.py` does work...)

As I am using an Ubuntu server, I eventually tried to install it with 
```console
$ sudo apt install python3-winrm
```
which solved the problem immediately.

TL;DR Try to install the `winrm` module throught your distribution's package manager and that might solve your problem.

Cheers.
